modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -125,6 +126,81 @@ For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
 ====
 +
+. If you use PXE UEFI, perform the following actions:
+.. Provide the `shimx64.efi` and `grubx64.efi` EFI binaries and the `grub.cfg` file that are required for booting the system.
+
+** Extract the necessary EFI binaries by mounting the {op-system} ISO to your host and then mounting the `images/efiboot.img` file to your host:
++
+[source,terminal]
+----
+$ mkdir -p /mnt/iso
+----
++
+[source,terminal]
+----
+$ mkdir -p /mnt/efiboot
+----
++
+[source,terminal]
+----
+$ mount -o loop rhcos-installer.x86_64.iso /mnt/iso
+----
++
+[source,terminal]
+----
+$ mount -o loop,ro /mnt/iso/images/efiboot.img /mnt/efiboot
+----
+
+** From the `efiboot.img` mount point, copy the `EFI/redhat/shimx64.efi` and
+`EFI/redhat/grubx64.efi` files to your TFTP server:
++
+[source,terminal]
+----
+$ cp /mnt/efiboot/EFI/redhat/shimx64.efi
+----
++
+[source,terminal]
+----
+$ cp /mnt/efiboot/EFI/redhat/grubx64.efi.
+----
++
+[source,terminal]
+----
+$ umount /mnt/efiboot
+----
++
+[source,terminal]
+----
+$ umount /mnt/iso
+----
+
+** Copy the `EFI/redhat/grub.cfg` file that is included in the {op-system} ISO to your TFTP server.
+
+.. Edit the `grub.cfg` file to include arguments similar to the following:
++
+----
+menuentry 'Install Red Hat Enterprise Linux CoreOS' --class fedora --class gnu-linux --class gnu --class os {
+	linuxefi rhcos-<version>-live-kernel-<architecture> coreos.inst.install_dev=/dev/sda coreos.live.rootfs_url=http://<HTTP_server>/rhcos-<version>-live-rootfs.<architecture>.img coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign
+	initrdefi rhcos-<version>-live-initramfs.<architecture>.img <2>
```

Comment:
Does the <2> call-out lead to something? Making sure I didn't miss it.

Reply:
Removed